### PR TITLE
add Conduit_lwt_cmdliner for a standard way to retrieve a `mode`

### PR DESCRIPTION
**Pull request by avsm**

### CODE REVIEW

File: opam [hunk 1]
```diff
@@ -22,6 +22,9 @@ depends: [
   "uri"
   "cstruct" {>="1.0.1"}
   "ipaddr" {>="2.5.0"}
+  "cmdliner"
+  "logs"
+  "fmt"
```
Comment (rgrinberg) [inline opam]: Where is `fmt` being used?

Comment (avsm) [inline opam]: There seems to be a bug in the Fmt_cli meta files (or some other combination) that doesn't pull in the right cmx files and causes a link error if all three aren't specified.  I'll leave this open until I can repro to create an upstream issue.

Comment (dbuenzli) [inline opam]: Do you have the link error ?

Comment (dbuenzli) [inline opam]: Also I don't see why you add logs I don't see it being used by this PR.

**Pull request closed without merging**